spec/helpers/finance_details_helper_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe FinanceDetailsHelper, type: :helper do
```

Comment:
Following on from my comment about the fact a helper already exists, I think it means we can remove this test